examples/OpenTelemetry/Exporter/ConsoleExporter/Program.cs

```diff
@@ -59,7 +59,7 @@ private static async Task MainAsync(string[] args)
                 {
                     try
                     {
-                        var x = await session.ExecuteAsync(new SimpleStatement("SELECT * FROM system.local"));
+                        var x = await session.ExecuteAsync(new SimpleStatement("SELECT * FROM system.local WHERE key='local'"));
```

Comment:
Since this information is not going anywhere lets pull only key:
```suggestion
                        var x = await session.ExecuteAsync(new SimpleStatement("SELECT key FROM system.local WHERE key='local'"));
```